ext/dba/dba_lmdb.c

```diff
@@ -56,6 +64,14 @@ DBA_OPEN_FUNC(lmdb)
 		return FAILURE;
 	}
 
+	if (mapsize > 0) {
```

Comment:
`mapsize` greater 0

Reply:
Yes, that's deliberate. If `mapsize==0` that means use the default mapsize, so there is no need to call `mdb_env_set_mapsize()`.